### PR TITLE
shaarli: on_install function

### PR DESCRIPTION
**Pull request by phylophyl**
please read https://github.com/freedombox/Plinth/issues/344

### CODE REVIEW

File: plinth/modules/shaarli/views.py [hunk 1]
```diff
@@ -28,6 +28,9 @@
 from plinth import package
 from plinth.modules import shaarli
 
+def on_install():
+"""Notify that the service is now enabled."""
+shaarli.service.notify_enabled(None, True)
 
 @package.required(['shaarli'])
```
Comment (jvalleroy) [inline plinth/modules/shaarli/views.py]: One more modification is needed. Line 35 should be:
```
@package.required(['shaarli'], on_install=on_install)
```
The first "on_install" is an optional parameter for the @package.required function. And the 2nd "on_install" is a reference to the function that you defined above.

Comment (phylophyl) [inline plinth/modules/shaarli/views.py]: From my account, it is still line 32
http://i.imgur.com/bAI44xd.png

+def on_install(): reference is not displayed
i am not sure then if i need to copy again +def on_install():

Comment (jvalleroy) [inline plinth/modules/shaarli/views.py]: @phylophyl I'm not sure I follow...where is that screenshot from? Also...why doesn't it have the on_install function that you added in this pull request?

**Comment by jvalleroy** [comment]
I've merged this with some additional changes.

In the future, I'd recommend avoiding Github's built-in editor for modifying code files. Because it seems like it doesn't provide syntax checking, or give an opportunity for testing the change.

**Comment by phylophyl** [comment]
yes sorry, it was difficult to get into this process, i will redo a test helping on another bug, it aims at proposing to translators a reward (learn coding with freedombox team)
which editor would you recommend?

**Comment by jvalleroy** [comment]
@phylophyl I use Emacs, but there are many good alternatives:
https://wiki.python.org/moin/PythonEditors
https://wiki.python.org/moin/IntegratedDevelopmentEnvironments

**Pull request merged**